src/main/java/ChatResponseRespone.java

```diff
@@ -0,0 +1,29 @@
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class ChatResponseRespone {
+    private List<Choice> choices;
+
+    public List<Choice> getChoices() {
+        return choices;
+    }
+
```

Comment:
The @Data annotation includes @Getter, @Setter, @ToString, and @EqualsAndHashCode. Using @Data may introduce unnecessary methods and increase the class complexity.